### PR TITLE
chore: add tooltips for node balances

### LINKED ISSUE #283: Add tooltips on all cards on node page
A user asked - what is receiving capacity? I don't understand it, I thought I could receive an infinite amount

### PR DESCRIPTION
**Pull request by im-adithya**
Fixes #283

Should I rename the Savings Balance to Onchain Balance?

Screenshot 2024-07-22 at 6 11 44 PM

### CODE REVIEW

File: frontend/src/screens/channels/Channels.tsx [hunk 1]
```diff
@@ -511,7 +511,19 @@ export default function Channels() {
...
+              <TooltipProvider>
+                <Tooltip>
+                  <TooltipTrigger>
+                    <div className="flex flex-row gap-2 items-center">
+                      Savings Balance
+                      <InfoIcon className="h-4 w-4 shrink-0 text-muted-foreground" />
+                    </div>
+                  </TooltipTrigger>
+                  <TooltipContent>
+                    You can use on-chain balance to open new lightning channels
```
Comment (rolznz) [inline frontend/src/screens/channels/Channels.tsx:523]:
```suggestion
                    Your savings balance (on-chain balance) can be used to open new lightning channels. When channels are closed, funds from your channel will be returned to your savings balance.
```

File: frontend/src/screens/channels/Channels.tsx [hunk 2]
```diff
@@ -554,7 +566,17 @@ export default function Channels() {
...
-              Spending Balance
+              <TooltipProvider>
+                <Tooltip>
+                  <TooltipTrigger>
+                    <div className="flex flex-row gap-2 items-center">
+                      Spending Balance
+                      <InfoIcon className="h-4 w-4 shrink-0 text-muted-foreground" />
+                    </div>
+                  </TooltipTrigger>
+                  <TooltipContent>Your lightning wallet balance</TooltipContent>
```
Comment (rolznz) [inline frontend/src/screens/channels/Channels.tsx:577]:
```suggestion
                  <TooltipContent>Your spending balance is the funds on your side of your channels, which you can use to make lightning payments.</TooltipContent>
```

File: frontend/src/screens/channels/Channels.tsx [hunk 3]
```diff
@@ -584,7 +606,19 @@ export default function Channels() {
...
+              <TooltipProvider>
+                <Tooltip>
+                  <TooltipTrigger>
+                    <div className="flex flex-row gap-2 items-center">
+                      Receiving Capacity
+                      <InfoIcon className="h-4 w-4 shrink-0 text-muted-foreground" />
+                    </div>
+                  </TooltipTrigger>
+                  <TooltipContent>
+                    Shows how much bitcoin you can currently receive
```
Comment (rolznz) [inline frontend/src/screens/channels/Channels.tsx:618]:
```suggestion
                    Your receiving balance is the funds owned by your channel partner, which will be moved to your side when you receive lightning payments.
```

Comment (im-adithya) [reply frontend/src/screens/channels/Channels.tsx]: I took these tooltip descriptions from [here](https://www.figma.com/design/2IyilDewDrdNP3G1NixPYd/Alby-Hub?node-id=4200-59372&t=ToHa0bXLMk56hWbk-1), we don't need one-liners?

Comment (rolznz) [reply frontend/src/screens/channels/Channels.tsx]: How do the tooltips look now? I think there were some issues in the original copy, which I tried to resolve with my suggestions

**Review by reneaaron — APPROVED** [review]
utACK

**Pull request merged**